README.md

```diff
@@ -11,6 +11,20 @@ It is published online at: [docs.ubuntu.com/en/styleguide](https://docs.ubuntu.c
 The Vale linter operates from a series of rules. These are defined in individual YAML files, grouped into 'Styles'.
 This repository contains the Canonical set of rules, or the Canonical Style. 
 
+### Manual check
+
+To manually check your documentation with Vale rules use the following steps:
```

Comment:
```suggestion
To manually check your documentation with Vale rules complete the following steps:
```

Reply:
I feel like `use` works better in this case.